assets/component-facets.css

```diff
@@ -326,7 +328,7 @@
   align-items: flex-start;
 }
 
-.facets-layout-grid .visual-display-parent {
+.facets-layout-grid .facets__label {
```

Comment:
This is overwriting the styles being applied by the `.facet-checkbox` declaration. That's fine to overwrite them... the weird part is that `.facet-checkbox` is setting the display, flex-direction, etc to begin with.  Two things to change that could fix the alignment of list items with a checkbox:

1. Change `.facet-checkbox` on line 345 to `.facets__label`.  This is in line with a number of other changes you made that removes the focus on `facet-chackbox` to the labels.
2. Add `align-items: center` to that declaration. 

This would fix a misalignment that's most visible in the mobile's drawer view

![](https://screenshot.click/28-26-0hs29-fvcsu.png)

You can also see it in the vertical layout; this is what it looks like with the fix applied

![](https://screenshot.click/28-28-nty9q-x1ab1.png)

Reply:
+1

Reply:
I talked with @NathanPJF about this. If the text wraps, then the checkbox (but not the checkmark) would get centered. We can fix the checkmark as well, but in either case, we decided having the checkbox centered looked worse, so we won't make this change. I can do the rename, but not the alignment.

![image](https://github.com/Shopify/dawn/assets/1019769/3f0e9521-dfed-4916-b9f3-4b54bfee27b8)